lib/counter.js

```diff
@@ -116,6 +124,14 @@ class Counter {
 	}
 }
 
+const reset = function() {
+	this.hashMap = {};
```

Comment:
Might be a good idea to call reset in the constructor instead of having duplicate code

Reply:
Ok